ui/src/layouts/BuildPublication.tsx

```diff
@@ -304,7 +303,7 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
         [checkRequiredApproval, isReadyToPreview, store]
     );
 
-    const coAuthorsToApprove = useMemo(() => CheckCoAuthorsToApprove(store).ready, [CheckCoAuthorsToApprove, store]);
+    const hasCoAuthors = useMemo(() => publicationHasCoAuthors(store).ready, [publicationHasCoAuthors, store]);
```

Comment:
not really sure if we need a useMemo just to check if the publication has coAuthors